src/requested_interactions.inc

```diff
@@ -64,14 +64,14 @@
   logical :: add_nutau_emission_epannihil = .true.
   logical :: add_anutau_emission_epannihil = .true.
 
-  logical :: add_nue_emission_NNBrems = .false.
-  logical :: add_anue_emission_NNBrems = .false.
-  logical :: add_numu_emission_NNBrems = .true.
-  logical :: add_anumu_emission_NNBrems = .true.
-  logical :: add_nutau_emission_NNBrems = .true.
-  logical :: add_anutau_emission_NNBrems = .true.	
+  logical :: add_nue_emission_bremsstrahlung = .false.
+  logical :: add_anue_emission_bremsstrahlung = .false.
+  logical :: add_numu_emission_bremsstrahlung = .false.
+  logical :: add_anumu_emission_bremsstrahlung = .false.
+  logical :: add_nutau_emission_bremsstrahlung = .false.
+  logical :: add_anutau_emission_bremsstrahlung = .false.	
 
-  logical :: add_nue_emission_weakinteraction_ecap = .true.
+  logical :: add_nue_emission_weakinteraction_ecap = .false.
```

Comment:
please make the pull request default to what was here before (both here and above with `logical :: add_nue_absorption_on_A = .false.`